doc/replacements/ens_palette_methods.xml

```diff
@@ -0,0 +1,31 @@
+<docstrings>
+
+<override namespace="ensight.objs.ENS_PALETTE.set_range_to_over_time_minmax">
+<signature>[]</signature>
```

Comment:
@randallfrank if the signature and parnames are empty, will it get them from the stubs?

Reply:
@kecolburn I just chatted with Randy. Since the signature and paramnames are optional and your intent is to not replace them, you need to just remove the tags, literally removing line 4 and 5. Otherwise the signature and paramnames will become empty

Reply:
I will also make this point in the README.md file.  The tags: <signature>, <paramnames>, <description> and <code> are all optional.  If you are ok with the auto-generated bits, there is no need to include them in the XML at all.  Sorry, I was not clear on that point.

Reply:
@randallfrank if you make the change in this PR we can merge it together